harfshapedfa/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "harfshapedfa"
```

Comment:
Jokes aside, I do think this needs a different name; we should make it clear that this isn't a HarfBuzz or skrifa project, and have something a little less confusing